### PR TITLE
Updates to course info section for OSPR-2017

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3838](https://openedx.atlassian.net/browse/DOC-3838)

OSPR-2017 ([platform PR 16638](https://github.com/edx/edx-platform/pull/16638)) updates the **Course Info** page on the instructor dashboard to provide the correct values. This PR updates the docs, which previously had to explain a mistake (the value for Course Name was really the course run).

### Date Needed (optional)

13 Dec 2017

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @pomegranited 
- [ ] Subject matter expert: 
- [x] Doc team review (copy edit?): @edx/doc
- [ ] Product review:
- [x] Partner support: @JAAkana
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### Sandbox (optional)

* LMS: https://pr16638.sandbox.opencraft.hosting/
* Studio: https://studio-pr16638.sandbox.opencraft.hosting/

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Review by grantgoodmanedX — APPROVED** [review]
👍  Looks good to me

File: en_us/shared/manage_live_course/course_data.rst [hunk 1]
```diff
@@ -29,13 +29,44 @@ To view this course data, follow these steps.
...
+      You can change the way that your course name appears in the LMS by using
+      Studio. To change the course name, open the course in Studio, select
+      **Settings**, select **Advanced Settings**, and then change the value for
+      the **Course Display Name** policy key. **EdX does not recommended that
+      you do this if your course is live.**
+
+   * **Course Run**: Specified when you create the course. Becomes part of the
+     course URL, and cannot be changed.
+   * **Course Number**: Specified when you create the course. Becomes part of
+     the course URL, and cannot be changed.
```
Comment (JAAkana) [inline en_us/shared/manage_live_course/course_data.rst]: I don't read code, but the comments in the pull request refer to:
Course Number: now uses the proper [display_number_with_default](https://github.com/edx/edx-platform/blob/fabccf6163f4f8fe68467d09afefcda2c0405e00/common/lib/xmodule/xmodule/course_module.py#L1311-L1319) property

My expectation would be that Course Number is specified when you created the course. Becomes part of the course URL. URL cannot be changed, but you can change how the course number appears in the LMS by using Studio / Settings / Advanced Settings / Course Number Display String

Comment (srpearce) [reply en_us/shared/manage_live_course/course_data.rst]: @pomegranited Can you confirm @JAAkana 's expectation above?

Comment (pomegranited) [reply en_us/shared/manage_live_course/course_data.rst]: @srpearce Confirmed!  Thanks for that clarification @JAAkana .

**Review by pomegranited — APPROVED** [review]
:+1:

- [x] I read through the code, and checked it against https://github.com/edx/edx-platform/pull/16638
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] Includes documentation

File: en_us/shared/set_up_course/studio_add_course_information/creating_new_course.rst [hunk 2]
```diff
@@ -120,4 +120,56 @@ team privileges.
...
+In the LMS, your course title changes to the value that you specify in the
+**Course Display Name** field. The URL for your course does not change.
+
+.. _Change the Course Number in the LMS:
+
+===================================
+Change the Course Number in the LMS
+===================================
+
+To change the way that your course title appears in the LMS, follow these
```
Comment (catong) [inline en_us/shared/set_up_course/studio_add_course_information/creating_new_course.rst]: "title" -> "number"

File: en_us/shared/set_up_course/studio_add_course_information/creating_new_course.rst [hunk 3]
```diff
@@ -120,4 +120,56 @@ team privileges.
...
+steps.
+
+.. note::
+ EdX does not recommended that you do this if your course is live.
+
+#. Open the course in Studio
+#. On the **Settings** menu, select **Advanced Settings**.
+#. In the **Course Number Display String** field, enter the title that you
```
Comment (catong) [inline en_us/shared/set_up_course/studio_add_course_information/creating_new_course.rst]: "title" -> "number"

**Comment by pomegranited** [comment]
Thank you @srpearce !

**Pull request merged**